### PR TITLE
Added instructions for ubuntu

### PR DESCRIPTION
**Pull request by Mohammad-Yasser**
While setting up the bot locally in ubuntu 18.04, I faced some problems. I used the instructions I added to solve these problems.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,20 @@ Now all dependencies need to be installed. TLE uses [Poetry](https://poetry.eust
...
+If you are using ubuntu with older versions of python, then do the following:
+```bash
+apt-get install python3.7-venv
+python3.7 -m pip install poetry
+python3.7 -m poetry install
+```
+
+TLE also depends on cairo and pango for graphics and text rendering, which you need to install through your package manager. For ubuntu, the relevant packages are
+```bash
+apt-get install libcairo2-dev libgirepository1.0-dev libpango1.0-dev pkg-config python3-dev gir1.2-pango-1.0
```
Comment (algmyr) [inline README.md:42]: Is `gir1.2-pango-1.0` a needed dependency. I can't remember it from when I helped setup another server, but maybe I missed it.

Comment (Mohammad-Yasser) [reply README.md:42]: I didn't need it to set it on my local PC which is Ubuntu 18.04, but I needed it when I set it up on Amazon EC2 instance using Ubuntu 18.04. :man_shrugging:

Comment (algmyr) [reply README.md:42]: interesting

**Comment by algmyr** [comment]
LGTM, thanks.

**Pull request merged**